cspell.json

```diff
@@ -1,31 +1,73 @@
 {
     "version": "0.2",
+    "ignorePaths": [
+        "eng/release/DiagnosticsReleaseTool",
+        "src/external",
+        "src/inc"
+    ],
     "words": [
+        "Bldr",
+        "Brotli",
+        "Browsable",
         "callstacks",
         "CLSID",
         "codespace",
         "coreclr",
         "darc",
+        "dbug",
+        "disambiguator",
         "discoverability",
+        "Distro",
         "dockerfiles",
+        "dylib",
         "ESRP",
+        "evented",
         "exfiltrate",
+        "fwlink",
         "gcdump",
         "gcdumps",
         "globbing",
         "globstar",
+        "hmac",
+        "HMACSHA",
+        "hresult",
+        "Hsts",
+        "Impls",
+        "JITID",
         "JWTs",
         "libc",
         "livemetrics",
         "minidump",
         "MSRC",
+        "ndjson",
         "newtonsoft",
+        "Notfication",
```

Comment:
Should this be included? It appears to just be a misspelling of "Notification"

Reply:
We'll probably pick up a new version of the header file this release so we can remove the exclusion at that time.